mixer/adapter/stackdriver/metric/bufferedClient.go

```diff
@@ -17,11 +17,16 @@ package metric
 import (
 	"context"
 	"io"
+	"strings"
 	"sync"
 	"time"
 
 	monitoringpb "google.golang.org/genproto/googleapis/monitoring/v3"
 
+	gprcstatus "google.golang.org/genproto/googleapis/rpc/status"
```

Comment:
curious: do we need this import, or can we use the `gogo/status` package that was already in the repo?

Reply:
It is not needed and removed after the newest commit.